pkg/template/core/starlark_value.go

```diff
@@ -57,6 +57,17 @@ func (e StarlarkValue) AsInt64() (int64, error) {
 	return 0, fmt.Errorf("expected starlark.Int")
 }
 
+// Accept either int or float starlark values.
```

Comment:
```suggestion
// AsFloat64 converts a Starlark number (either int or float) to the corresponding Go double-precision float.
```

Our lint config grandfather's in code we wrote prior to our starting to use the linter. This is why existing code is not GoDoc'ed, but the linter is complaining about this function.

It's a little aggressive, but on balance it's leading to a more readable codebase.